astropy/coordinates/sites.py

```diff
@@ -0,0 +1,138 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+Observatories accessible by the `sites` module originate from the IRAF
+Observatory Database, and are stored in ``data/observatories.json``.
+Longitudes are listed with positive to the West.
```

Comment:
Good catch - I'm going to be switching to the newer astroplan one in a commit I send up shortly, anyway.